### PR TITLE
ch8 jlis implemented. straight forward implementation TLE

### PR DESCRIPTION
**Pull request by riceluxs1t**
1. implemented ch8 problem JLIS (https://algospot.com/judge/problem/read/JLIS)
2. straight forward DP solution translated from JMB TLE (it's expected some python solutions with correct asymptotic runtime complexities TLE)

### CODE REVIEW

File: ch8/jlis.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+def jlist(index_A, index_B):
```
Comment (riceluxs1t) [inline ch8/jlis.py:1]: sample comment1

File: ch8/jlis.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+	for next_B in xrange(index_B + 1, m):
+		if max_element < B[next_B]:
+			ret = max(ret, jlist(index_A, next_B) + 1)
+
+	cache[(index_A + 1, index_B + 1)] = ret
+	return ret
+
+
+C = input()
```
Comment (riceluxs1t) [inline ch8/jlis.py:23]: sample comment2.

File: ch8/jlis.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+for _ in xrange(C):
```
Comment (riceluxs1t) [inline ch8/jlis.py:25]: sample comment3.

I/O code should be consistent in style.
e.g. read in a line of integers = map(int, raw_input().strip().split())

File: ch8/jlis.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+		if max_element < A[next_A]:
+			ret = max(ret, jlist(next_A, index_B) + 1)
+
+	for next_B in xrange(index_B + 1, m):
+		if max_element < B[next_B]:
+			ret = max(ret, jlist(index_A, next_B) + 1)
+
+	cache[(index_A + 1, index_B + 1)] = ret
+	return ret
+
```
Comment (riceluxs1t) [inline ch8/jlis.py:21]: sample comment 4

this python code is almost line by line translation from the cpp code in the book

File: ch8/jlis.py (hunk 1 above)
Comment (sj43) [reply ch8/jlis.py:1]:
```suggestion
def jlist(index_A, index_B):
    testing
```

**Pull request merged**